src/storage/mod.rs

```diff
@@ -4875,7 +4875,7 @@ mod tests {
                 commit_ts,
                 version,
                 key.clone(),
-                Key::from_raw(b"z"),
+                Key::from_raw_maybe_unbounded(b"z"),
```

Comment:
What about passing `Some(Key::from_raw(b"z"))` in tests since we can make sure that it's not empty?

Reply:
Okay, get it.